### PR TITLE
feature: resolve first exercies

### PR DESCRIPTION
**Pull request by JuanCZer**
Complete first exercise

### CODE REVIEW

File: FirstExercises/index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (ericklugoj) [inline FirstExercises/index.html:7]: Rename the title

```suggestion
    <title>Excercise 1</title>
```

File: FirstExercises/index.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+// TODO
+//Change hola mundo to me la pela Rivas
+const title =  document.querySelector("h1").innerHTML = "me la pela Rivas";
+const title3 =  document.querySelector(".hola").innerHTML = "Te amo Lugo";
```
Comment (ericklugoj) [inline FirstExercises/index.js:4]: **new todo:**

change the title after 3 seconds

**Review by ericklugoj — CHANGES_REQUESTED** [review]
read comments

File: FirstExercises/index.html (hunk 1 above)
Comment (JuanCZer) [reply FirstExercises/index.html]: Donzo

File: FirstExercises/index.js (hunk 2 above)
Comment (JuanCZer) [reply FirstExercises/index.js]: Donzo moderoquer

**Review by ericklugoj — APPROVED** [review]
Excellent 🚀

**Pull request merged**